docs/DeveloperGuide.adoc

```diff
@@ -1206,6 +1206,100 @@ Expected: No meme is liked. Error details shown in the status message. Status ba
 2. Dislike a meme when there are memes listed +
 Similar to like. Command is `dislike`.
 
+=== Suggest / Auto-complete commands
+. Suggest and auto-complete command words
+.. Prerequisites: Weme is in any tab, for illustration purpose we choose meme tab.
+Since this is for command suggestions, please do not press kbd:[enter] to execute the command while typing the input.
+..  Test case: `a` +
+    Expected: +
+    `add: adds a meme to Weme. +
+    archive: archive a meme by index. +
+    archives: list all archived memes.`
+..  Test case: `a` + kbd:[tab] +
+    Expected text inside command box: `add{nbsp}` +
+    Expected text inside result box: +
+    `add: adds a meme to Weme. Parameters: p/PATH [d/DESCRIPTION] [t/TAG]... +
+Example: add p/C:\Users\username\Downloads\funny_meme.jpg d/Popular Meme among SoC Students  t/funny`
+..  Test case: `arkive` +
```

Comment:
The typo is on purpose?

Reply:
Yes, to show that typo can be auto-corrected by suggestion.